staging/src/k8s.io/kubectl/pkg/cmd/apply/applyset.go

```diff
@@ -86,6 +87,36 @@ func (a ApplySet) Validate() error {
 	return utilerrors.NewAggregate(errors)
 }
 
+func (a *ApplySet) LabelsForMember() map[string]string {
+	return map[string]string{
+		"applyset.k8s.io/part-of": a.ID(),
```

Comment:
nit: I'd make this key a constant, especially since we're going to need to need the value in other places

Reply:
Good point - I'll do that in the next PR!